### PR TITLE
Header download & documentation buttons (animated click & outline)

### PR DESCRIPTION
**Pull request by nicholedwight**
Fixing this issue: https://github.com/shahednasser/sbuttons/issues/133

I added two buttons to the header under the subtitle text, the download button is an animated click with a white background and black text, the documentation link button has a white background with black text and border.

I also added css to index.css for overriding the button colors to be white (along with comments for other contributors), I'm more than happy to make a permanent white class for all the buttons though!

✨

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -68,6 +68,8 @@ <h1 class="noselect">sButtons</h1>
         <div class="subtitle">
           Simple buttons you can use easily for your next project.
         </div>
+        <a href="#" id="downloadGithubRawButton" class="sbtn click-btn btn-rounded">Download</a>
+        <a href="https://github.com/shahednasser/sbuttons" target="_blank" class="sbtn outline-btn btn-rounded black-btn">Documentation</a>
```
Comment (shahednasser) [inline index.html:72]: I noticed you added a `black-btn` class here, but I don't think it's necessary.

**Review by shahednasser — CHANGES_REQUESTED** [review]
I like your work! I think it would be best if both buttons were click buttons because the first button looks much better. But we don't want them to be the same color, so maybe use `yellow-btn` for one of them but make the other white? Or if you have another color in mind (other than the ones we are already using) you can use it too! Make sure that the color that stands out is the color for the download button, as we need the focus to be on it.

Someone is already working on adding a white button, so hopefully when they're done we'll be able to replace the override with that class!

File: index.html (hunk 1 above)
Comment (nicholedwight) [reply index.html:72]: You got it! I'll make these changes :)

**Comment by shahednasser** [comment]
Looks great! Just one final change: I feel like it'll look better if the button's font was larger. Can you change the font-size to 1.3rem?

**Comment by nicholedwight** [comment]
Coming right up!

**Comment by shahednasser** [comment]
Awesome! Thank you so much.

@all-contributors please add @nicholedwight for code

**Pull request merged**